Hakawai/HKWExternalMentionConstants.h

```diff
@@ -0,0 +1,22 @@
+//
+//  HKWExternalMentionConstants.h
+//  Hakawai
+//
+//  Created by Chen Yuan on 3/23/21.
+//  Copyright © 2021 LinkedIn. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface HKWExternalMentionConstants : NSObject
+
+/**
+ Character set for at symbols to prepend to mention text, including @, ＠ (Japanese), for the ease of library consumers.
+ */
```

Comment:
if you could style this as:

Character set for at symbols to prepend to mention text, including @, ＠ (Japanese)
NOTE: This is for the ease of library consumers, not for use inside the library